static/app/components/editableText.tsx

```diff
@@ -132,121 +131,60 @@ function EditableText({
   }
 
   return (
-    <Wrapper>
-      <InnerWrapper ref={innerWrapperRef} isDisabled={isDisabled} isEditing={isEditing}>
-        {isEditing ? (
-          <InputWrapper isEmpty={isEmpty} data-test-id="editable-text-input">
-            <StyledField inline={false} flexibleControlStateSize stacked>
-              <StyledInput
-                name={name}
-                ref={inputRef}
-                value={inputValue}
-                onChange={handleInputChange}
-              />
-            </StyledField>
-            <InputLabel>{inputValue}</InputLabel>
-          </InputWrapper>
-        ) : (
-          <React.Fragment>
-            <Label
-              onClick={isDisabled ? undefined : handleEditClick}
-              ref={labelRef}
-              data-test-id="editable-text-label"
-            >
-              <InnerLabel>{inputValue}</InnerLabel>
-            </Label>
-            {!isDisabled && <StyledIconEdit />}
-          </React.Fragment>
-        )}
-      </InnerWrapper>
+    <Wrapper ref={innerWrapperRef} isDisabled={isDisabled} isEditing={isEditing}>
+      {isEditing ? (
+        <InputWrapper isEmpty={isEmpty} data-test-id="editable-text-input">
+          <StyledInput
+            name={name}
+            ref={inputRef}
+            value={inputValue}
+            onChange={handleInputChange}
+          />
+          <InputLabel>{inputValue}</InputLabel>
+        </InputWrapper>
+      ) : (
+        <React.Fragment>
+          <Label
+            onClick={isDisabled ? undefined : handleEditClick}
+            ref={labelRef}
+            data-test-id="editable-text-label"
+          >
+            <InnerLabel>{inputValue}</InnerLabel>
+            {!isDisabled && <IconEdit />}
```

Comment:
should we set the icon's cursor to `pointer`?

Reply:
Since the inner label + icon is clickable to start editing I added the `cursor: pointer` to the label wrapper :) Looks good